src/clusterfuzz/_internal/datastore/data_handler.py

```diff
@@ -1535,6 +1535,15 @@ def get_testcase_variant(testcase_id, job_type):
   return variant
 
 
+def get_all_testcase_variants(testcase_id):
+  """Get all testcase variant entities based on testcase id."""
+  variants_query = data_types.TestcaseVariant.query(
+      data_types.TestcaseVariant.testcase_id == testcase_id)
+  if not variants_query:
```

Comment:
I think this will never be the case. can omit.

Reply:
Done.